lib/signaling/v2/peerconnection.js

```diff
@@ -1159,4 +1183,46 @@ function shouldStopTransceiver(sdp, transceiver) {
   return false;
 }
 
+/**
+ * Perform certain updates after an SDP negotiation is completed.
+ * @param {PeerConnectionV2} pcv2
+ * @returns {void}
+ */
+function negotiationCompleted(pcv2) {
+  if (pcv2._isRTCRtpSenderParamsSupported) {
+    updateEncodingParameters(pcv2);
+  }
+}
+
+/**
+ * Update the RTCRtpEncodingParameters of all active RTCRtpSenders.
+ * @param {PeerConnectionV2} pcv2
+ * @returns {void}
+ */
+function updateEncodingParameters(pcv2) {
+  const { maxAudioBitrate, maxVideoBitrate } = pcv2._encodingParameters;
+
+  const maxBitrates = new Map([
+    ['audio', maxAudioBitrate || 0],
+    ['video', maxVideoBitrate || 0]
+  ]);
+
+  pcv2._peerConnection.getSenders().filter(sender => sender.track).forEach(sender => {
+    const maxBitrate = maxBitrates.get(sender.track.kind);
+    const params = sender.getParameters();
+
+    if (isFirefox) {
+      params.encodings = [{ maxBitrate }];
+    } else {
+      params.encodings.forEach(encoding => {
+        encoding.maxBitrate = maxBitrate;
+      });
+    }
+
+    sender.setParameters(params).catch(error => {
```

Comment:
@makarandp0 I don't think we should do that, since there is no point calling the function when the API is not supported. Also, it is clearer to read this way.